Gaijinizer patch/CommonEvents_121.txt

```diff
@@ -31,42 +33,47 @@ up again.#MTLed
 そんな状態になっても『快楽が勝ってる』って、
 いったいどんな気分なんだろ？
 > CONTEXT: CommonEvents_121/128/Dialogue/enemy2
-I want to pant, but is my throat dehydrated with
-dehydration already? Even in such a state,
-"pleasure overwrites it", What is it like?#MTLed
+You're trying to gasp but your throat's too
+parched from dehydration? But even in that kind of
+situation, pleasure overrides everything? Just
+what kind of mental state are you in?#Ron
```

Comment:
Flow, and a rewrite of the second sentence to make it make more sense (feel free to comment if it's too far away from the original):
>_**Are you trying to gasp but your throat's too
parched from dehydration? How does it feel to
have pleasure override all of your thoughts
and feelings?**_

(This is a hard one to translate, but I've done my best...)